stable/datadog/values.yaml

```diff
@@ -540,6 +540,14 @@ clusterAgent:
   #
   useHostNetwork:  # true
 
+  ## @param dnsConfig - list of objects - optional
+  ## specify dns configuration options for datadog cluster agent containers e.g ndots
+  dnsConfig:
+  #  dnsConfig:
```

Comment:
This indented `# dnsConfig` line is misleading, some people may think they need to double it.
Also as this field content should be a valid Kubernetes dnsConfig, you should link the Kubernetes doc:
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config

Reply:
thanks for the review @vboulineau 
I agree and I changed my docs and suggested default value as you requested

please review again